build/maven.go

```diff
@@ -93,6 +96,11 @@ func (mm *MavenModule) SetExtractorDetails(localExtractorPath, extractorPropsdir
 	return mm
 }
 
+func (mm *MavenModule) SetOutputWriter(outputWriter io.Writer) *MavenModule {
```

Comment:
Let's remove the output value to be consistent with the other setters
```suggestion
func (mm *MavenModule) SetOutputWriter(outputWriter io.Writer) {
```